### ISSUE TITLE
Unable to set flatpak theme

### ISSUE THREAD

**Issue by RedAuburn** [largestgithubuseronearth/addwater#15]
**To Reproduce**
Have Firefox installed, both system (dnf) and flatpak.
Change preference from base to flatpak
try to apply the theme
get a message "Installation failed."

**Expected behavior**
theme is set for the flatpak install of firefox.

**Please include the log file from the day it happened**
```
harry@fedora:~/Documents$ flatpak run dev.qwery.AddWater 
/var/home/harry/.var/app/dev.qwery.AddWater/cache/add-water already exists. Skipped.
/var/home/harry/.var/app/dev.qwery.AddWater/cache/add-water/downloads already exists. Skipped.
/var/home/harry/.var/app/dev.qwery.AddWater/cache/add-water/logs already exists. Skipped.
All paths initialized.
-------------------------
[INFO] addwater.apps.firefox.firefox_details — 01:01 || Setting Firefox data path: /var/home/harry/.mozilla/firefox/
[INFO] addwater.apps.firefox.firefox_details — 01:01 || Valid path. Done.
[INFO] addwater.backend — 01:01 || Backend created for Firefox
[INFO] addwater.components.online — 01:01 || Checking for updates...
[INFO] addwater.components.online — 01:01 || no update available
[INFO] addwater.preferences — 01:01 || Preferences Window activated
[WARNING] addwater.preferences — 01:01 || Autofind paths disabled
[INFO] addwater.preferences — 01:01 || User specified path: /var/home/harry/.var/app/org.mozilla.firefox/.mozilla/firefox/
[INFO] addwater.apps.firefox.firefox_details — 01:01 || Setting Firefox data path: /var/home/harry/.var/app/org.mozilla.firefox/.mozilla/firefox/
[INFO] addwater.apps.firefox.firefox_details — 01:01 || Valid path. Done.
[INFO] addwater.backend — 01:02 || beginning installation...
[INFO] addwater.components.install — 01:02 || kicking off install...
[ERROR] addwater.components.install — 01:02 || Install failed. can't find theme path OR profile path.
[INFO] addwater.backend — 01:02 || install process completed
```

**Technical Details (please complete the following information):**
 - Distribution: Fedora 41 Silverblue
 - Add Water version: 1.1

 - Browser: Firefox
 - Browser Package: Flatpak (base is also installed)
 - Browser Version: 131.0.3

**Comment by largestgithubuseronearth** [comment]
Sorry about that. After looking into it, it seems the Package switcher isn't properly resetting the interface. So when you click apply, it's still trying to install to the base package but failing because the paths don't match up.

I will work on this tonight and push a fix tomorrow morning.

Thanks for taking the time to report this.

**Comment by largestgithubuseronearth** [comment]
Status update: The process is a bit more involved than I expected and will require refactoring a couple of important pieces of code. Expect an update within the next couple of days.

Sorry again, and thanks for your patience.

**Comment by RedAuburn** [comment]
Thank you for looking into it! :)

**Comment by largestgithubuseronearth** [comment]
I've been working to rework of how the app handles data failures and switching between package types. But in the meantime, I'm pushing a fix for the bug that would overwrite your chosen package when the app launches. 

This just means you'll need to set the package format to Flatpak in the Preferences, and then re-launch the app. But after 
that it should work as expected.  Ideally, it would just refresh the normal interface but that's turned out to be more complicated than I hoped. Sorry about that. I'll keep on working but I don't want to keep you waiting on this major issue.

The update (v.1.1.2) should appear in the next few hours. Please let me know if the issue persists or another issue occurs after this update.

Thank you for your patience.
